packages/backend-modules/assets/lib/getCropDimensions.js

```diff
@@ -0,0 +1,37 @@
+module.exports = (crop) => {
+  if (!crop) {
+    return {
+      x: null,
+      y: null,
+      width: null,
+      height: null,
+    }
+  }
+  // /[^0-9]/ regex: match non-numeric values
+  // turns '10x20y100w10h' into ["10", "20", "100", "10", ""] and removes the last value
+  const [_x, _y, _width, _height] = crop.split(/[^0-9]/).slice(0, -1)
+  const x = _x ? Math.ceil(Math.abs(_x)) : null
```

Comment:
Depeding on how you split regex before, `-` won't be recognized as a digit and thus `Math.abs` is useless.

Also, would apply `Math.ceil` after you validated varaibles is not nan or not of type number starting L:18.